rust/src/table_properties.rs

```diff
@@ -67,3 +67,5 @@ pub const SET_TRANSACTION_RETENTION_DURATION: &str = "delta.setTransactionRetent
 pub const TARGET_FILE_SIZE: &str = "delta.targetFileSize";
 /// The target file size in bytes or higher units for file tuning. For example, 104857600 (bytes) or 100mb.
 pub const TUNE_FILE_SIZES_FOR_REWRITES: &str = "delta.tuneFileSizesForRewrites";
+/// Invariants defined on the table
+pub const INVARIANTS: &str = "delta.invariants";
```

Comment:
I don't think this belongs here. INVARIANTS isn't a table-level property, it's at the field level in the schema. See an example here:

https://github.com/delta-io/delta-rs/blob/f3176e0b9fa350cefd462297e4abe861f280612a/python/tests/pyspark_integration/test_write_to_pyspark.py#L57-L68